node.gypi

```diff
@@ -24,6 +24,7 @@
     },
     'force_load%': '<(force_load)',
   },
+  'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
```

Comment:
Does this need to be in an `OS` conditional as it was in `common.gypi`?

Reply:
AFAICT GYP knows what to do, so no need to exclude win and mac.
I'll investigate the CI output

Reply:
> I'll investigate the CI output

* On freebsd (which uses clang)
```
ccache c++ -o /out/Release/obj.target/node_lib/src/callback_scope.o ... -m64 -Wall -Wextra -Wno-unused-parameter
```
* On Mac (which is clang bu via XCODE):
```
/usr/local/bin/ccache c++ -o out/Release/obj.target/node_lib/src/callback_scope.o ../src/callback_scope.cc ... -Wall -Wendif-labels -W -Wno-unused-parameter
```
(only the old set, this run was before I moved it)
* On Windows
https://gist.github.com/refack/aa6d6bac65bcd0ab4187aa86bfc27fce
No mention of any of the cflags